src/main/java/uk/gov/cslearning/catalogue/domain/Course.java

```diff
@@ -42,7 +42,7 @@
     private String preparation;
 
     @NotNull
-    private Visibility visibility;
```

Comment:
We will probably want to make an update to the elastic data set instead of just setting defaults